dictionaries/PropertyMap.php

```diff
@@ -260,7 +260,7 @@
         'field_count' => 'int',
         'host_info' => 'string',
         'info' => 'string',
-        'insert_id' => 'int',
+        'insert_id' => 'int|string',
```

Comment:
Wouldn’t  it make sense to have `numeric-string` here?

Reply:
Probably. @muglug?

Reply:
Actually, this is one of the many places where I would like to see an `int-string` type. `numeric-string` is just way too broad to be relevant in real life, IMHO.